fenix/app/src/main/java/org/mozilla/fenix/settings/logins/SavedLoginsViewModel.kt

```diff
@@ -0,0 +1,26 @@
+package org.mozilla.fenix.settings.logins
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+/**
+ * [ViewModel] to store and get query from SavedLoginScreen
+ */
+class SavedLoginsViewModel : ViewModel() {
```

Comment:
We are generally against using new `LiveData` / `ViewModel` instances as they bring a level of complexity we don't need on top of the current `Store`s which [is already wrapped in a `ViewModel`](https://searchfox.org/mozilla-mobile/rev/2d9dd7b862d87ac539a52b9b0d99160b61745dae/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/components/StoreProvider.kt#18-29) with the help of the [StoreProvider.get](https://searchfox.org/mozilla-mobile/rev/2d9dd7b862d87ac539a52b9b0d99160b61745dae/fenix/app/src/main/java/org/mozilla/fenix/settings/logins/fragment/SavedLoginsFragment.kt#72) call.
We should look into using the [searchedForText](https://searchfox.org/mozilla-mobile/rev/2d9dd7b862d87ac539a52b9b0d99160b61745dae/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/settings/logins/LoginsFragmentStore.kt#78) property from `LoginsFragmentState`.